bigchaindb/web/views/metadata.py

```diff
@@ -0,0 +1,50 @@
+"""This module provides the blueprint for some basic API endpoints.
+
+For more information please refer to the documentation: http://bigchaindb.com/http-api
+"""
+import logging
+
+from flask_restful import reqparse, Resource
+from flask import current_app
+
+from bigchaindb.backend.exceptions import OperationError
+from bigchaindb.web.views.base import make_error
+
+logger = logging.getLogger(__name__)
+
+
+class MetadataApi(Resource):
+    def get(self):
+        """API endpoint to perform a text search on transaction metadata.
```

Comment:
This new HTTP API endpoint must be documented in the page about the HTTP API, similar to the documentation about the asset text search endpoint: https://docs.bigchaindb.com/projects/server/en/master/http-client-server-api.html#assets

Reply:
Sure.